### PR TITLE
Marcel projections (pybbda integration)

### PR DESCRIPTION
**Pull request by bdilday**
This PR implements marcel projections, based on the code in [pybbda](https://github.com/bdilday/pybbda)

### CODE REVIEW

File: pybaseball/analysis/projections/marcels/marcels_base.py [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+Module defining base class for marcel projections. Marcel projections are defined as the minimal projections system
+that does anything reasonable. They
+
+* make a rate projection based on last 3 seasons
+    * projection is weighted average of performance plus a regression to the mean
+* make a playing time projection
+
+for batters the weights are 5, 4, 3 and for pitchers 3, 2, 1
+
+The original description of Marcels is in this post by Tome Tanog
```
Comment (schorrm) [inline pybaseball/analysis/projections/marcels/marcels_base.py:11]: Typo

File: docs/lahman.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Pull data from [Sean Lahman's database](http://www.seanlahman.com/baseball-archive/statistics/), also hosted by [Chadwick Bureau on GitHub](https://github.com/chadwickbureau/baseballdatabank) -- our new source -- using the following functions:
 
 ```python
-from pybaseball.lahman import *
+from pybaseball.datasources.lahman import *
```
Comment (schorrm) [inline docs/lahman.md:6]: I don't love this indirection, especially because lahman is user-facing (which `datasources.lahman` isn't)

Comment (bdilday) [reply docs/lahman.md:6]: yes - pycharm IDE did this when I moved the file and I didn't catch it

**Comment by bdilday** [comment]
Note that this PR uses pandas nullable integer type and so requires pandas v1+ @schorrm @jldbc

**Comment by schorrm** [comment]
Okay, this will need to have the setup.py changed

**Comment by schorrm** [comment]
Lahman is usable as was previously?

**Comment by bdilday** [comment]
> Lahman is usable as was previously?

no. `from pybaseball import batting` would work as before, but `from pybaseball import lahman` would not. I feel like this is a design decision that's outside the scope of this PR. I will put it back how it was.

File: tests/pybaseball/analysis/projections/marcels/test_marcels_batting.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import pytest
+
+
+class TestMarcelBatting:
+    def test_batting_projections_season(self):
+        md = MarcelProjectionsBatting()
+        md.projections(2020)
+
+    @pytest.mark.parametrize(
+        "season, expected", [(2020, 36), (2019, 38), (2018, 41), (2017, 34), (2004, 42)]
```
Comment (TheCleric) [inline tests/pybaseball/analysis/projections/marcels/test_marcels_batting.py:12]: ❤️

File: tests/pybaseball/test_team_batting.py [hunk 4]
```diff
@@ -4,8 +4,8 @@
 import pytest
 import requests
 
-from pybaseball.team_batting import _FG_TEAM_BATTING_URL, team_batting
-
+from pybaseball import team_batting
+from pybaseball.team_batting import _FG_TEAM_BATTING_URL
```
Comment (TheCleric) [inline tests/pybaseball/test_team_batting.py:8]: Looks like another PyCharm import change here

**Comment by bdilday** [comment]
> Okay, this will need to have the setup.py changed

done.

File: pybaseball/__init__.py [hunk 5]
```diff
@@ -65,3 +66,5 @@
 from .team_game_logs import team_game_logs
 from .plotting import plot_stadium
 from .plotting import spraychart
+
+NULLABLE_INT = Int32Dtype()
```
Comment (schorrm) [inline pybaseball/__init__.py:70]: Would this make more sense in utils?

Comment (bdilday) [reply pybaseball/__init__.py]: yes, good suggestion. done.

**Comment by schorrm** [comment]
Last little thing: can you remove `from pandas import Int32Dtype` from the top of __init__?

**Comment by schorrm** [comment]
Great! Thank you!

**Pull request merged**